test/upgradeability.test.js

```diff
@@ -25,7 +25,8 @@ const getFlattenedContractFromMasterBranch = _ => {
   sed -i ${platform === 'darwin' ? ' \'\' -e' : ''} 's/contract PToken is/contract \
   PTokenMaster is/g' .${CONTRACT_FROM_MASTER_PATH} && \
   cd ../ && \
-  npm run compile`
+  npm run compile && \
+  sleep 1`
```

Comment:
Worth adding a comment as to why this sleep is here.

Reply:
In the end I removed it.
Interestingly, it was the 05-ptoken.test.js test breaking upgradeability.test.js...
In particular `getContractFactory('PReceiver')`. I replaced it with `getContractFactory('contracts/test-contracts/PReceiver.sol:PReceiver')` and now the problem is gone.
Probably specifying the name only breaks an internal cache or something like that.

Reply:
Good find in that case!